### ISSUE TITLE
Automated hyperparameter tuning and logging

### ISSUE THREAD

**Issue by pranshu-raj-211** [pranshu-raj-211/human-action-recognition#1]
Keras tuner and wandb for now, following the template, later switch to pytorch and sweep to improve automation.

Key hyperparameters that I need to focus on:
- Learning rate
- Window Size (data)
- Convolutional layer kernel and stride
- regularization
- Optimizers (RMSProp, AdaGrad, Adam, AdaDelta)
- number of conv layers
